src/main/java/org/springframework/data/redis/core/RedisKeyValueAdapter.java

```diff
@@ -393,6 +394,109 @@ public Long doInRedis(RedisConnection connection) throws DataAccessException {
 		return count != null ? count.longValue() : 0;
 	}
 
+	public void update(final PartialUpdate<?> update) {
```

Comment:
Old index references remain in index-helper index `(entity):(key):idx` when updating/deleting `@Indexed` properties